test/e2e-common.sh

```diff
@@ -197,9 +197,9 @@ function test_task_creation() {
   sidecars:
   - image: quay.io/chmouel/go-rest-api-test
     name: go-rest-api
-    volumeMounts:
-    - name: fixtures
-      mountPath: /fixtures
+    envFrom:
+    - configMapRef:
+        name: fixtures
```

Comment:
Is this needed then?
The point of the volume mount was to be able to use `file` in the fixture config.
If we inline it instead, I don't see the advantage of using this now, unless we change the fixture tool to load fixture responses from env.